samples/bluetooth/mesh/light_switch/prj.conf

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General configuration
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_HWINFO=y
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+CONFIG_LOG=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_LOG_PRINTK=y
+
+CONFIG_DK_LIBRARY=y
+CONFIG_BT_MESH_DK_PROV=y
+
+# Bluetooth configuration
```

Comment:
Maybe we could do something similar for BT MESH samples?
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1594/files#diff-25775b8be1637aad0917e4de7644d3c8R7

Reply:
I have a task in my backlog for going through the config and see if I can add more `imply`s throughout. I'd rather deal with the samples as part of that task :)